modules/pytket-qsharp/pytket/extensions/qsharp/backends/azure_quantum.py

```diff
@@ -34,7 +33,8 @@
 
 from pytket.extensions.qsharp.backends.config import QSharpConfig
 from pytket.extensions.qsharp.backends.common import _QsharpBaseBackend
-from pytket.extensions.qsharp.qsharp_convert import tk_to_qsharp
```

Comment:
We should get rid of `tk_to_qsharp` altogether if possible, but I see it is still used in the `QSharpBaseBackend`. Can we make that use QIR too?